vehicle_data.json

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
Yeah, this output is cool. Usually our format for API data are nested arrays, so for example, all the makes would each be elements in an array where there is a key of `name` or `make`, then there's a nested hash structure inside of that. We'll learn more about that in mod 3. Great work!